Sources/SIL/SIL.swift

```diff
@@ -290,7 +295,9 @@ public enum Enforcement {
 public enum FunctionAttribute {
     case alwaysInline
     case differentiable(_ spec: String)
+    case dynamicallyReplacable
     case noInline
+    case readonly
     case semantics(_ value: String)
```

Comment:
Enum payloads don't need argument names or labels. 

`case semantics(String)` is totally fine.